tests/ops/qubit/test_parametric_ops.py

```diff
@@ -1854,3 +2017,14 @@ def test_pow_matrix(self, op, n):
 def test_control_wires(op, control_wires):
     """Test the ``control_wires`` attribute for parametrized operations."""
     assert op.control_wires == control_wires
+
+
+class TestEigvalCalculation:
```

Comment:
I don't think it is the best place for this test, you can simply add one after the isingXY tests, also you have to add multiple interfaces for the the tests (tf, torch and jax), it will solve your coverage problem. Also you can use `qml.math.eigvals` for agnostic function for eigenvalues  and `qml.math.allclose` to compare the results and not `Counter`.

Reply:
done!

Reply:
Hey @rmoyard, as the values are complex, there is no direct way to arrange them in ascending or descending order. I instead decided to verify them against the analytic formula for the eigenvalues derived from the matrix directly. This works perfectly.

Reply:
Nice @ankit27kh, I will give a last review round today to be sure that everything is fine 👍